### ISSUE TITLE
Add Calendar app

### ISSUE THREAD

**Issue by akshgpt7** [akshgpt7/MarvellOS#17]
A calendar app would be awesome to have.
It would display the calendar of every month for various years.
The `calendar` python module can directly be used to generate the dates and years, so just positioning and displaying them properly is the thing to take care of.

**Comment by kunal351411** [comment]
Can I work on this ?

**Comment by akshgpt7** [comment]
@kunal351411 sure, go for it!

**Comment by khushi0704** [comment]
@akshgpt7 please let me do that , I've already completed it 70% .
please assign it to me ! I've already completed with adding the calendar app on the apps windows and I'm only left with importing calendar function to it .
ASSIGN ME otherwise my efforts till now will go in vain
![Screenshot (52)](https://user-images.githubusercontent.com/64957861/101332616-89ec8e80-389b-11eb-91ca-0deded523af5.png)
![Screenshot (53)](https://user-images.githubusercontent.com/64957861/101332650-92dd6000-389b-11eb-9f54-c0b36dec17c4.png)

**Comment by kunal351411** [comment]
@khushi0704 no problem you can do it. Because I haven't started yet.

**Comment by akshgpt7** [comment]
@khushi0704 please comment on the issue before you start working on it next time. It helps avoid confusion and such clashes.

**Comment by khushi0704** [comment]
yes sure I'll keep that in mind . Sorry for the confusion ! @akshgpt7 
thanks @kunal351411

**Comment by kunal351411** [comment]
@akshgpt7 is this still not resolved ?

**Comment by akshgpt7** [comment]
@khushi0704 Is working on this, and has an open PR already.
